src/ngraph/runtime/cpu/pass/cpu_memory_optimization.cpp

```diff
@@ -62,8 +62,9 @@ bool runtime::cpu::pass::CPUMemoryOptimization::run_on_function(std::shared_ptr<
 {
     for (auto n : function->get_ordered_ops())
     {
-        if (auto concat = std::dynamic_pointer_cast<op::Concat>(n))
+        if (n->description() == "Concat")
```

Comment:
I'm not sure I understand the rationale behind this change. We go from using the proper RTTI mechanism to compare the description string? What is the rationale?

Reply:
On Mac, dynamic_pointer_cast would return a nullptr even though the op is Concat.

Reply:
Thanks, Amy. I had no idea. Is this a known issue on Mac side or some problem on our side? It looks weird to me that we use a string to do RTTI since it's not very efficient. Is this a common practice/workaround?

Reply:
http://www.russellmcc.com/posts/2013-08-03-rtti.html. This is a workaround.